src/businesses/service/BusinessesCrudService.ts

```diff
@@ -77,4 +94,22 @@ export default class BusinessesCrudService {
     }
   }
 
+  private mapUserBusinessPermissionsPromise(
```

Comment:
@mb-ca please fix this Sonar issue:
https://sonarcloud.io/project/issues?branch=feature%2FNon-Admin_Navigation_Permissions&id=channelape-typescript-sdk&resolved=false&severities=MAJOR&sinceLeakPeriod=true